release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3194,6 +3194,30 @@ $ oc adm release info 4.14.2 --pullspecs
 
 * Previously, when MetalLB Operator's controller restarted while having an IP assigned and unassigned load balancer services, it restarted with an empty internal state, which can break workloads. With this release, MetalLB's controller is modified to first process the services that already have an assigned IP. (link:https://issues.redhat.com/browse/OCPBUGS-16267[*OCPBUGS-16267*])
 
+* Previously, if you created an Operator group with same name as privileged cluster roles used by {product-title} and Kubernetes resources, Operator Lifecycle Manager (OLM) would overwrite the privileged cluster roles. With this fix, if you create an Operator group that conflicts with a privileged cluster role, OLM generates a unique cluster role name using the following syntax:
```

Comment:
I think I made a similar note on the 4.15 variation of the doc, but "privileged cluster roles" don't exist, IIRC the ClusterRoles we "protect" are the ones found in OpenShift by default.